hr_salary_rule_reference/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013 Savoir-faire Linux
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Salary Rule Reference',
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'category': 'Generic Modules/Human Resources',
+    'author': "Savoir-faire Linux,Odoo Community Association (OCA)",
```

Comment:
I think you should change the author below for new version:
```
'author': "<your name>"
           "Odoo Community Association (OCA)",
```

Reply:
As OCA guildelines (link below), I think we just need to put the original author and OCA, no need for the migrators